Source/core/Time.h

```diff
@@ -343,7 +358,13 @@ namespace Core {
         {
             return (static_cast<uint16_t>(time.tm_yday));
         }
+        inline bool IsValidDateTime(const uint16_t year, const uint8_t month, const uint8_t day, const uint8_t hour, const uint8_t minute, const uint8_t second, const uint16_t millisecond) const
```

Comment:
adding the inline here does not make much difference, better to add it to IsValidDate below

Reply:
removed from here, but not adding to IsValidaDate, since its definition is in CPP